GDAXSharp/Shared/Utilities/Extensions/ProductTypeExtensions.cs

```diff
@@ -18,5 +18,12 @@ public static Currency QuoteCurrency(this ProductType value)
 
             return (Currency)Enum.Parse(typeof(Currency), quoteCurrency);
         }
+
+        public static string ToDasherizedUpper(this ProductType orderType)
```

Comment:
shouldn't need this. you should be able to use the methods above here to get the currency in the format you are looking for

Reply:
In your webfeed branch I saw you created a function like this. so I somewhat copied it.